examples/protocol_buffers/car.proto

```diff
@@ -0,0 +1,11 @@
+syntax = "proto3";
+
+import "examples/protocol_buffers/tire.proto";
+
+message Car {
+  string make = 1;
+  string model = 2;
+  int32 year = 3;
```

Comment:
You might consider exercising your test with a message containing an int64. `pbjs` has a couple of modes for handing `int64` in JS and one of the requires that the `long` node_module be used. We should test that we integrated that scenario correctly

Reply:
thanks, good point